tools/image-tool/src/crane.rs

```diff
@@ -0,0 +1,41 @@
+use std::path::Path;
+
+use async_trait::async_trait;
+
+use crate::{cli::CommandLine, ImageTool, Result};
+
+pub struct CraneCLI {
+    pub(crate) cli: CommandLine,
+}
+
+#[async_trait]
+impl ImageTool for CraneCLI {
+    async fn pull_archive(&self, path: &Path, uri: &str, _cache: &Path) -> Result<()> {
```

Comment:
nit: this name isn't totally accurate since it pulls the extracted version right? maybe `pull_oci_image` would be better?